apps/arena/lib/arena/game_updater.ex

```diff
@@ -1528,6 +1504,10 @@ defmodule Arena.GameUpdater do
     real_damage = Player.calculate_real_damage(attacking_player, projectile.aditional_info.damage)
     crate = Crate.take_damage(crate, real_damage)
 
+    unless Crate.alive?(crate) do
+      send(self(), {:crate_destroyed, attacking_player.id, crate.id})
+    end
```

Comment:
This is done in multiple places. We should isolate this behavior to update every destroyed crate state in a single pass in only one place.